e2e-tests/cypress/integration/appeal-submission-appeal-site-address/appeal-submission-appeal-site-address.js

```diff
@@ -88,12 +88,12 @@ Then(
         break;
       case 'Address Line 1 has a limit of 60 characters':
         cy.confirmSiteAddressWasRejectedBecause(
-          'Building and/or street must be 60 characters or fewer',
+          'The first line of the address must be 60 characters or fewer',
         );
         break;
       case 'Address Line 2 has a limit of 60 characters':
         cy.confirmSiteAddressWasRejectedBecause(
-          'Building and/or street must be 60 characters or fewer',
+          'The first line of the address must be 60 characters or fewer',
```

Comment:
does this need to be the 'second line' of address?

Reply:
It does indeed - great spot, thanks :+1: